lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -19,120 +20,186 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
+
+Function CheckModified-File($url, $dest, $headers, $credentials, $timeout, $use_proxy, $proxy) {
```

Comment:
@jborean93 That's not what I want to do. The current module automatically uses the system's (or the users's) proxy, with credentials, if it is configured, and no `proxy_url` is provided. I would like to keep the existing behaviour as it was. That's why I cannot configure `$proxy` in this case.

I think it's fine like it is.